applications/dashboard/views/log/record.php

```diff
@@ -14,7 +14,7 @@ class="Info"><?php echo t('Edits and deletions are recorded here. Use &lsquo;Res
         echo anchor(t('Restore'), '#', ['class' => 'RestoreButton btn btn-primary']);
         echo anchor(t('Delete Forever'), '#', ['class' => 'DeleteButton btn btn-primary']);
         ?>
-        <?php PagerModule::write(['Sender' => $this, 'Limit' => 10, 'View' => 'pager-dashboard']); ?>
+        <?php PagerModule::write(['Sender' => $this, 'Limit' => $this->data('Limit'), 'View' => 'pager-dashboard']); ?>
```

Comment:
This is a nice change, but we've gone from always having a value for `Limit` here to potentially having nothing (e.g. `Limit` isn't set in the data array). Maybe we should include a default. Otherwise, I think we'll either be using whatever the default of `PagerModule` is (probably not 10) or the default of `data` (an empty string?) will lead to an error when rendering.

Reply:
I thought this was used as the default https://github.com/vanilla/vanilla/blob/7a85198092eb0b40e1577e97d7013c6350556fce/applications/dashboard/modules/class.pagermodule.php#L100 when no `$Limit` is provided.

Reply:
> I thought this was used as the default [...] when no $Limit is provided.

It is. That's what I was referring to here:

> Otherwise, I think we'll either be using whatever the default of PagerModule is (probably not 10) [...]

I'm pretty sure that default isn't going to be 10, which was the limit we were using here before. It looks like it might actually be 30. Was altering the default limit on this page an intended change?